examples/certs/make_keyprint.sh

```diff
@@ -2,21 +2,21 @@
 #
 # make_keyprint.sh
 #
-# based on a script by ullner
-# source: https://adc.svn.sourceforge.net/svnroot/adc/trunk/Source/scripts/keyp_keyprint_generation.sh
```

Comment:
Moreover the new source is https://gitlab.com/dcnf/adc-code/blob/master/Source/scripts/keyp_keyprint_generation.sh